exec.go

```diff
@@ -21,19 +19,19 @@ var execCommand = cli.Command{
 			Value: "",
 			Usage: "specify the pty slave path for use with the container",
 		},
+		cli.BoolFlag{
+			Name:  "tty, t",
+			Usage: "allocate a pseudo-TTY",
+		},
 	},
 	Action: func(context *cli.Context) {
-		config, err := loadProcessConfig(context.Args().First())
```

Comment:
ya do you think a flag? or?

On Fri, Jan 15, 2016 at 11:56 AM, Mike Brown <notifications@github.com>
wrote:

> In exec.go
> <https://github.com/opencontainers/runc/pull/479#discussion_r49897729>:
>
> >  	},
> >  	Action: func(context *cli.Context) {
> > -		config, err := loadProcessConfig(context.Args().First())
>
> Will probably need a path where there is an attempt to load a process
> config file for the exec. The reason for this is the way start uses the
> config.json file.. While there is no definitive requirement yet for the
> process config as a separate file in the opencontainers/specs, there is a
> high likelihood that this pattern in start of being able to point to a file
> name to say what the options were when the command was executed, will need
> to still be supported for high level containerd like users of runc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/479/files#r49897729>.
>